### DISCUSSION TITLE
Error calling IoT1ClickProjects.listProjects, Error calling OpsWorksCM.describeServers while connecting to AWS from Former2

### DISCUSSION THREAD

**Discussion by gollasandeepkumar** [iann0036/former2#369] [General]
Hello,

I am trying to connect my AWS account from Former2 website using AccessKey, SecretKey and SessionKey. I am seeing below errors when I click on Scan Account. It looks like AWS IoT 1-Click service will discontinue in December 2024. Could you please let me know if there is any alternative method to connect to my AWS account for generating IaC. 

Errors:
Your call to the AWS IoT 1-Click API failed. AWS IoT 1-Click service is not accepting new customers because AWS will discontinue the service on December 16, 2024. To learn more, refer to AWS IoT 1-Click documentation.

AWS OpsWorks CM is not accepting new customers. This command is therefore no longer supported.

Thanks
Sandeep

![Screenshot 2024-05-29 at 9 01 24 PM](https://github.com/iann0036/former2/assets/128870368/d9e25c1a-4d1b-4fd8-9714-d76bc740df50)

**Comment by iann0036** [comment]
Hi @gollasandeepkumar,

Thanks for bringing this to my attention. I've made a change to ignore these two deprecation errors now.

https://github.com/iann0036/former2/commit/641fa1ae73ced0619d0b3b021959c74e80102596

**Reply by gollasandeepkumar** [reply]
@iann0036 Thank you so much for your prompt response. The issue is now resolved, and I really appreciate your help with this.
